README.md

```diff
@@ -0,0 +1,23 @@
+# Unofficial flatpak for Claws-Mail
```

Comment:
I'm usually unhappy about files like this because they duplicate what's already in documentation. If some steps required guessing then we need to improve the docs instead of writing it here.

Reply:
I've hidden _Notes_ and _References_ section, to avoid exposing possibly outdated information.
You're suggesting removing the whole file right? If so, I can do that once the package is mature, i.e. all TODOs tackled and everything fine as far as everyone can tell. As for the _Unofficial_ status, can I indicate this somewhere?

Reply:
TODO is fine, I'm only concerned in build steps (etc).